src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldNotContainUnderscores.Fixer.cs

```diff
@@ -19,11 +27,50 @@ public sealed override FixAllProvider GetFixAllProvider()
             return WellKnownFixAllProviders.BatchFixer;
         }
 
```

Comment:
I think you don't need C# and VB sub-types of `IdentifiersShouldNotContainUnderscoresFixer`. You can just make `IdentifiersShouldNotContainUnderscoresFixer` sealed and add the C# and VB export code fix attribute to it.